wfc3tools/wf3cte.py

```diff
@@ -105,12 +36,8 @@ def wf3cte(input, output=None, parallel=True, verbose=False, log_func=print):
         - a single filename (``iaa012wdq_raw.fits``)
         - a Python list of filenames
         - a partial filename with wildcards (``*raw.fits``)
-        - filename of an ASN table (``*asn.fits``)
         - an at-file (``@input``)
 
-    output : str, default=None
-        Name of the output FITS file.
-
     parallel : bool, default=True
```

Comment:
Will the other command line arguments be added in the future? If so, when and if not, why?